src/ui/contextmenu.rs

```diff
@@ -38,9 +38,12 @@ impl ContextMenu {
         track: Track,
     ) -> Modal<Dialog> {
         let mut list_select: SelectView<Playlist> = SelectView::new().autojump();
+        let current_user_id = spotify.current_user().unwrap().id;
```

Comment:
Can I be sure that the current_user always is available here or should I handle this differently?

Reply:
Should be fine. The UI doesn't show if ncspot can't create a valid session. Maybe it could be cached within Spotify as it's unlikely to change during runtime to avoid repeated API requests.

Reply:
I saw the library struct has a private field with the user_id, maybe making that field public and using it is a better strategy?

Reply:
Yeah, that would be one approach.